lib/services/elasticsearch.js

```diff
@@ -1384,21 +1431,15 @@ class ElasticSearch extends Service {
     { refresh, timeout } = {})
   {
     const
-      esRequest = {
-        index: this._getESIndex(index, collection),
-        body: { ids: { values: [] } },
-        scroll: '30s',
-        refresh,
-        timeout
-      },
+      query = { ids: { values: [] } },
       toGet = [],
-      toDelete = [],
       partialErrors = [];
 
-    if (ids.length > this.kuzzle.config.limits.documentsWriteCount) {
+    // @todo should be done in clientAdapter
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)